app/controllers/api/v2/interfaces_controller.rb

```diff
@@ -32,12 +32,12 @@ def show
 
       def_param_group :interface_attributes do
         #common parameters
-        param :mac, String, :required => true, :desc => N_("MAC address of interface")
-        param :ip, String, :required => true, :desc => N_("IP address of interface")
-        param :type, InterfaceTypeMapper::ALLOWED_TYPE_NAMES, :required => true, :desc => N_("Interface type, e.g: bmc")
-        param :name, String, :required => true, :desc => N_("Interface's DNS name")
+        param :mac, String, :desc => N_("MAC address of interface. Required for managed interfaces on bare metal.")
+        param :ip, String, :desc => N_("IP address of interface")
+        param :type, InterfaceTypeMapper::ALLOWED_TYPE_NAMES, :desc => N_("Interface type, e.g: bmc. Default is %s") % InterfaceTypeMapper::DEFAULT_TYPE.name
```

Comment:
Interpolating like this won't work, as you'll pass the string "Interface type... default: managed" through to apipie, which when it comes to translate later, won't find that string.  (Because it's the %s version that's extracted.)

Look at config/initializers/apipie.rb and its "substitutions" thing to see how to do this.